tests/serverpod_test_client/lib/src/http/http_status_io.dart

```diff
@@ -0,0 +1 @@
+export 'dart:io' show HttpStatus;
```

Comment:
Suggestion: Only use our definition of `HttpStatus`.

Since we now have our own definition of `HttpStatus` we no longer need to import it from `dart:io`. Then we can remove all the conditional import statements and just have a single definition for it.